no-gettext-template eslint rule

arski opened the pull request:
@udemy/team-f moving the new rule from https://github.com/udemy/website-django/pull/35799 into js-tooling.

Would you be so kind to do the publishing please - the django PR is basically ready to merge, I can do so as soon as new version is available. Thanks!

arski committed:
no-gettext-template eslint rule

kevinzang commented:
@arski can you add some unit tests? See https://github.com/udemy/js-tooling/blob/master/packages/eslint-plugin-udemy/rules/import-blacklist/tests.js for example. Run the tests via `yarn test`.

arski committed:
Add test

arski commented:
@kevinzang I knew you'd ask me that :)) done! (PS not really sure where to run them from as I don't have yarn in any of the js-tooling directories :X)

arski committed:
Fix config

kevinzang reviewed (approved):
@arski LGTM!

Commit:
Publish

 - babel-polyfill-udemy-website@9.0.10
 - babel-preset-udemy-website@11.0.7
 - eslint-config-udemy-basics@8.0.4
 - eslint-config-udemy-website@12.0.9
 - eslint-plugin-udemy@9.0.2
